Source/CoreExtensions/Private/BlueprintLibraries/CoreExtMathBlueprintLibrary.cpp

```diff
@@ -1,5 +1,26 @@
 #include "BlueprintLibraries/CoreExtMathBlueprintLibrary.h"
 
+void UCoreExtMathBlueprintLibrary::GetPointsOnRectangleEdges( TArray< FVector2D > & out_point_table, const float height, const float width, const float point_spacing )

```

Comment:
`point_table` (we don't use the `out_` prefix

Reply:
I saw this in the function just under "mine" .. it mislead me :P